osu.Game/OsuGame.cs

```diff
@@ -34,18 +37,51 @@ public override void Load()
             base.Load();
 
             Add(new Drawable[] {
-                MainMenu = new MainMenu(),
+                intro = new Intro(),
                 Toolbar = new Toolbar
                 {
-                    OnHome = delegate { MainMenu.MakeCurrent(); },
+                    OnHome = delegate { MainMenu?.MakeCurrent(); },
                     OnSettings = delegate { Options.PoppedOut = !Options.PoppedOut; },
                     OnPlayModeChange = delegate (PlayMode m) { PlayMode.Value = m; },
+                    Alpha = 0.001f //fixes invalidation fuckup

```

Comment:
in case i forget, you can reproduce by making this line `Alpha = 0` and adjusting the toolbar's `FadeIn` to 500ms or longer.